posts/inside-rust/0000-00-00-const-eval-roadmap-and-skill-tree.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Constant evaluation roadmap"
+author: Oli Scherer
+team: the const eval working group <https://github.com/rust-lang/const-eval/>
+---
+
+Y'all probably noticed that const eval finally got loops, mutable local variables and branches. With those features existing, we were able to make a lot of basic arithmetic functions `const fn `. Things like `checked_div` needs to be able to check that the divisor is not zero and return `None` in that case. These features have been asked for a lot, and while it may not appear so, these were the low hanging fruit.
```

Comment:
I think you have an extra space here.

```suggestion
Y'all probably noticed that const eval finally got loops, mutable local variables and branches. With those features existing, we were able to make a lot of basic arithmetic functions `const fn`. Things like `checked_div` needs to be able to check that the divisor is not zero and return `None` in that case. These features have been asked for a lot, and while it may not appear so, these were the low hanging fruit.
```